inst/instance/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eo pipefail
+
+heroku login
+
+heroku create lightmlflow-testing
+heroku addons:create heroku-postgresql:hobby-dev --app lightmlflow-testing
```

Comment:
how is this script used? `deploy.sh` depends on the `lightmlflow-testing` app existing, so evidently this should be run before then (but only once?)

Reply:
I just ran it as a one-off to get the app set up, but I still can't figure out why building in GHA is failing so I might just deploy it once from local and kick the can